projects/marketing-site/components/Team/CoreTeam.tsx

```diff
@@ -30,14 +30,14 @@ const people = [
 
 export default function CoreTeam() {
   return (
-    <div className="bg-gray-900 rounded-lg">
+    <div className="dark:bg-gray-900 bg-gray-200 rounded-lg">
```

Comment:
see how the background color looks in light mode with bg-gray-50 

i think the section should have more contrast between the background colors and the text, in light mode